checks/phpcs.php

```diff
@@ -113,6 +113,7 @@ protected function phpcs_result_to_warnings( $result ) {
 
 				if ( current_user_can( 'edit_plugins' ) ) {
 					$edit_link   = sprintf(
+						/* translators: 1: URL to edit 2: Title of the Anchor 3: Aria Label 4: Text of the Anchor */
```

Comment:
@bordoni I don't think this is needed here since the line beneath it isn't being translated